atk/sys/src/lib.rs

```diff
@@ -1819,6 +1824,11 @@ extern "C" {
     //=========================================================================
     pub fn atk_layer_get_type() -> GType;
 
+    //=========================================================================
+    // AtkLive
+    //=========================================================================
+    pub fn atk_live_get_type() -> GType;
```

Comment:
Needs a since annotation otherwise things will break, I opened a PR for it upstream so :/

Reply:
Also as there is a new type, maybe it is worth adding to the safe bindings as well & it would need a since annotation there as well

Reply:
Gonna add the new type (even though it doesn't seem to be used anywhere?) and add the missing cfg.